### DISCUSSION TITLE
Filter im Gruppenmonitor

### DISCUSSION THREAD

**Discussion by v0lli** [brutella/hkknx-public#293] [Ideas]
in der aktuellsten Version [2.8.0-b2] läuft zumindest bei mir das Fenster mit den erfassten Ereignissen sehr schnell voll und scrollt automatisch nach unten für die aktuellsten Werte. Nun ist es ziemlich schwer wieder zur Filtereingabe zu gelangen. 
Vorschläge:
- Vielleicht kann man den Inhalt unterhalt des Filters scrollen lassen, und den Filter zur Eingabe bestehen lassen.
- mein Reflex war mit der ESC Taste das scrollen zu beenden (hat nicht funktioniert - wäre aber vielleicht praktisch).
- auch fände ich es praktisch Filtertexte zu kombinieren z.B. mit && oder || um besser filtern zu können.

**Comment by brutella** [comment]
Sobald du nach oben scrollst, sollte die Anzeigen nicht mehr automatisch nach unten scrollen. Funktioniert das bei dir nicht?

Finde deine beiden anderen Vorschläge interessant. Werde ich mir genauer ansehen. Danke für das Feedback.

**Comment by v0lli** [comment]
das nach oben scrollen funktioniert wenn nichts im Filter eingetragen ist. 
Sobald ich beginne einen Filter einzutippen z.B. für eine Adresse 1.1.6 und ich Adressen von 1.1.1 bi 1.1.250 habe - trifft die Regel, markiert den getroffenen Eintrag und dann kann ich nicht mehr nach oben scrollen um den Filter anzupassen / fertig zu schreiben. 
Ich komme dann gar nicht Mehr an das Filterfenster, und muss den Browsertab schließen.

**Reply by brutella** [reply]
Das Scrolling-Problem sollte in Version [2.8.0-b4](https://github.com/brutella/hkknx-public/releases/tag/2.8.0-b4) behoben sein.

**Comment by v0lli** [comment]
Super, das scrolling ist jetzt behoben - Danke!
Vielleicht passt das besser in eine neues Ticket - oder zu dem #295 ?
jetzt ist mir bei der besseren Übersichtlichkeit des Logs noch folgendes Aufgefallen:
Die Datenpunkte DPST-9-20 und DPST-9-21 sind Spannung und Stromstärke - werden bei mir aber im Log als °C angezeigt. 
Evtl. nur die Einheit am "Value" falsch?


Time | Priority | Source | Source | Communication Object | Destination | Destination | Type | Data Type | Value | Data
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
2024-01-13 08:01:12.304 | Low | 1.1.12 | Enertex KNX Power Supply 960-V2 | sVoltageGO | 7/4/1 | PSD_Spannung - Messung | Write | DPST-9-20 | 29634.56 °C | 5d a7
2024-01-13 08:01:12.336 | Low | 1.1.12 | Enertex KNX Power Supply 960-V2 | sCurrentGO | 7/4/2 | PSD_Stromstärke - Messung | Write | DPST-9-21 | 642.56 °C | 2f d8
2024-01-13 08:01:12.368 | Low | 1.1.12 | Enertex KNX Power Supply 960-V2 | sAUXCurrentGO | 7/4/3 | PSD_Stromstärke_AUX - Messung | Write | DPST-9-21 | 143.28 °C | 1e ff

![Screenshot 2024-01-13 at 08 11 38](https://github.com/brutella/hkknx-public/assets/26031457/b9f1572c-a75a-47b2-b87d-17452b0c3238)

**Reply by brutella** [reply]
Die Datenpunkte sollten jetzt in Version [2.8.0-b7](https://github.com/brutella/hkknx-public/releases/tag/2.8.0-b7) richtig angezeigt werden.

**Reply by v0lli** [reply]
ja, werden korrekt angezeigt - Danke!
